apps/nuxt3-ssr/components/CohortList.vue

```diff
@@ -0,0 +1,255 @@
+<script setup lang="ts">
+import type { IFilter } from "~/interfaces/types";
+
+const route = useRoute();
+const router = useRouter();
+const config = useRuntimeConfig();
+const pageSize = 10;
+
+useHead({ title: "Cohorts" });
```

Comment:
setting globals from components is not going to work, this whole idea of moving the pages stuff into the components is breaking the modular design

Reply:
I don't think I created this; originally CohortList was a page but now we render it on different places. However, if we use 'all' as a special filter then we can move this back to a page.